internal/mode/static/manager.go

```diff
@@ -213,15 +213,41 @@ func StartManager(cfg config.Config) error {
 		return fmt.Errorf("cannot register status updater: %w", err)
 	}
 
-	if err = mgr.Add(createTelemetryJob(cfg)); err != nil {
+	logger := cfg.Logger.WithName("telemetryJob")
```

Comment:
@sjberman Does it matter if I declare logger and exporter outside of `CreateTelemetryJobWorker`? I saw that you had put them inside of the method, but I thought that, especially for logger as its used in the function and when creating telemetryJob, it would be fine to declare them outside.

Reply:
One thing I wanted to do is to simplify StartManager because this function has a lot going on. I created the separate function to initialize the telemetry job to simplify the code a bit. I would still like that pattern here.

Following this pattern will allow you to have all of the added logic here in that separate function, and you can still return the runnable.